packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -40,6 +41,64 @@ const LOADPANEL_MESSAGE_CLASS = 'dx-loadpanel-message';
 const MESSAGE_SELECTOR = '.' + LOADPANEL_MESSAGE_CLASS;
 const LOADPANEL_CONTENT_CLASS = 'dx-loadpanel-content';
 const LOADPANEL_PANE_HIDDEN_CLASS = 'dx-loadpanel-pane-hidden';
+const LOADINDICATOR_CLASS = '.dx-loadpanel-indicator';
+
+QUnit.module('Accessibility', () => {
+    QUnit.test('showIndicator option should properly change from true to false during runtime', function(assert) {
+        const loadPanel = $('#loadPanel').dxLoadPanel({
+            visible: true,
+            showIndicator: true,
+        }).dxLoadPanel('instance');
+
+        assert.strictEqual(loadPanel.option('showIndicator'), true, 'showIndicator option is true');
+
+        loadPanel.option('showIndicator', false);
+
+        assert.strictEqual(loadPanel.option('showIndicator'), false, 'showIndicator option is false');
```

Comment:
Hm, what's the meaning of this test? I don't see how it can be red

Reply:
hi anton It is to replicate the scenario which showIndicator option will be set to false then i will check if it will properly set to false.

Reply:
but okay if this is not enough maybe i can check if the load indicator exist within the load panel element

Reply:
Still don't get why it's necessary. It's like test
```javascript
let a = 2;
assert.strictEqual(a, 2);
```
:)

Reply:
[7c89496](https://github.com/DevExpress/DevExtreme/pull/27080/commits/7c89496fe5ea9a3a695c7dcce3717057deeca65c)